### DISCUSSION TITLE
Table - Fixed column 移动端滑动体验优化

### DISCUSSION THREAD

**Discussion by yesongling** [rsuite/rsuite#1835] [General]
问题描述：
项目里面用到的表格字段比较多，在移动端需要固定住某个字读，其他被溢出隐藏的字段通过左右滑动查看
刚好Table组件的Fixed Column功能能够满足，但是在移动端出现了Table左右滑动与页面上下滚动冲突的情况。

希望能调整一下Table组件左右滑动的逻辑。

**Comment by simonguo** [comment]
以下是我手机上看到的效果，能够正常滚动，不清楚你说的冲突是什么。 另外不推荐在移动端上使用 Table 设计。

https://user-images.githubusercontent.com/1203827/129190294-ce9d0f42-fa99-4c5d-ac20-b764640df5f1.mov
